media_driver/cmake/linux/media_feature_flags_linux.cmake

```diff
@@ -28,6 +28,7 @@ bs_set_if_undefined(Encode_VDEnc_Supported "yes")
 # The opposite is full-feature which permits some features only have binaries.
 # The default build is full-feature build so set Full_Open_Source_Support as "no".
 bs_set_if_undefined(Full_Open_Source_Support "no")
+bs_set_if_undefined(FULL_OPEN_BUILD "no")
```

Comment:
What's the difference between Full_Open_Source_Support and FULL_OPEN_BUILD? If this PR will be merged, this will become a mess!!! We **must** agree on approach first and _refactor_ rather than introduce "fast" solution.

Reply:
+1 on @dvrogozh comment.... only one clear option should be enough